app.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const app = express();
+const topicsController = require("./controllers/topics.controller.js")
+
+
+app.get("/api/topics", topicsController.getTopics)
+
+app.all("*", (req, res) => {
+    res.status(404).send({msg: "404 Route Not Found"})
+})
+
+app.use((err, req, res, next) => {
```

Comment:
As you currently don't need any custom error handling in this endpoint, it's good practise not to include code you don't yet need and isn't used, so you could remove this.